### PR TITLE
Completion of the stage of translation of the course into Russian

### PR DESCRIPTION
**Pull request by blademoon**
Finalizing the translation phase. Units 6 and 7 have been translated. #109

### CODE REVIEW

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:91]: Looks like a verb is missing. "обманом" (заставить?) перевести

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:94]:
```suggestion
	`generate_kwargs={"task": "transcribe", "language": "es"}`
```

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:116]:
```suggestion
Мы увидим, что перевод более или менее совпадает (вы можете проверить это с помощью Google Translate), за исключением нескольких
```

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:175]: This sentence hasn't been translated.

**Review by MKhalusova — CHANGES_REQUESTED** [review]
Massive kudos for working on the translation! I found a couple of minor things to fix, after that it should be ready to merge once [chapter 5](https://github.com/huggingface/audio-transformers-course/pull/123) is merged.

File: chapters/ru/chapter6/pre-trained_models.mdx
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx]: Yes, indeed, corrected.

Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx]: Yes, indeed, corrected.

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Corrected.

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Corrected)

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Done.

File: chapters/ru/chapter7/speech-to-speech.mdx
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Added a translation of the missing sentence. Thank you for your attention Maria.

**Comment by blademoon** [comment]
@MKhalusova :rocket: Поехали)))

**Review by MKhalusova — COMMENTED** [review]
Overall great translation! Thank you for working on it. I have left a few minor nits.
We have also recently updated the TTS section with Bark, so if you'd like to include it in the translation as well, that would be awesome!

File: chapters/ru/chapter6/pre-trained_models.mdx
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx:1]: Thanks Maria. Let's continue the discussion in  our issues [#109](https://github.com/huggingface/audio-transformers-course/issues/109).

Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx:1]: Let's take this translation all the way to the end. Then we will add new material.

File: chapters/ru/chapter6/pre-trained_models.mdx
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx:184]: I think you're right. Let's leave the model name untranslated.

File: chapters/ru/chapter6/pre-trained_models.mdx
Comment (MKhalusova) [reply chapters/ru/chapter6/pre-trained_models.mdx:1]: Sure, feel free to add Bark in a separate PR

**Review by MKhalusova — APPROVED** [review]

**Pull request merged**
